app/builders/subscriber_auth_email_builder.rb

```diff
@@ -25,23 +25,11 @@ def call
   attr_reader :subscriber, :destination, :token
 
   def subject
-    "Confirm your email address"
+    manage_your_subscriptions_subject
```

Comment:
Can we put the subject text directly into this method? Do we need the extra method call?

Reply:
yeah can move it back, thought i'd leave it in case we followed up with further changes?